cwl/msi.cwl

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: cwl:v1.0
+
+#msisensor:
+#    msisensor msi \
+#    -d msi_sites
+#    -n normal_bam
+#    -t tumor_bam
+#    -tumor_sample_name tumor_sample_name
+#    -normal_sample_name normal_sample_name
+#    -o ${ return inputs.tumor_sample_name +"."+inputs.normal_sample_name+".msi.txt" }
+#
+
+class: CommandLineTool
+baseCommand:
+- "msisensor"
+- "msi"
+
+
+requirements:
+  DockerRequirement:
+    dockerPull: mskcc/msisensor:0.2
+  InlineJavascriptRequirement: {}
+  ResourceRequirement:
+    ramMin: 16
+    coresMin: 2
+
+doc: |
+  Run msisensor on tumor-normal bams to differentiate MSI (microsatellite instable) samples from MSS (microsatellite stable) ones
+
+inputs:
+  d:
```

Comment:
@timosong @svural can we change these to use more descriptive labels like "`microsatellites_file`", "`normal_bam_file`", etc..? Using these letter labels makes it really confusing to read especially in the workflow sections.